pkg/jtracer/jtracer.go

```diff
@@ -109,9 +111,17 @@ func otelResource(ctx context.Context, svc string) (*resource.Resource, error) {
 	)
 }
 
+func defaultGRPCOptions() []otlptracegrpc.Option {
+	var options []otlptracegrpc.Option
+	if strings.ToLower(os.Getenv("OTEL_EXPORTER_OTLP_INSECURE")) == "true" {
```

Comment:
https://github.com/jaegertracing/jaeger/blob/883d68193f36e717f1f0ec80b26b6f7a0718c5b4/examples/hotrod/pkg/tracing/init.go#L74

Reply:
Added the negation of the above logic to add `Insecure` option.